editor/components/colors/utils.js

```diff
@@ -15,7 +15,7 @@ import { find, kebabCase } from 'lodash';
  */
 export const getColorValue = ( colors, namedColor, customColor ) => {
 	if ( namedColor ) {
-		const colorObj = find( colors, { slug: namedColor } );
+		const colorObj = ( find( colors, { slug: namedColor } ) || find( colors, { name: namedColor } ) );
 		return colorObj && colorObj.color;
```

Comment:
@jorgefilipecosta @Luehrsen, how do you feel about adding a deprecation note in here so we could inform about the breaking changes introduced? I wasn't aware of that fact, but it seems like a good idea to keep JS console noisy for 2-3 releases to help them migrate to the latest API structure.